webapp/locales/de.json

```diff
@@ -143,27 +148,43 @@
             "successDelete": "Social-Media gelöscht. Profil aktualisiert!"
         },
         "muted-users": {
-            "name": "Stummgeschaltete Benutzer",
-            "explanation": {
-                "intro": "Wenn ein anderer Benutzer von dir stummgeschaltet wurde, dann passiert folgendes:",
-                "your-perspective": "In deiner Beitragsübersicht tauchen keine Beiträge der stummgeschalteten Person mehr auf.",
-                "search": "Die Beiträge von stummgeschalteten Personen verschwinden aus deinen Suchergebnissen."
-            },
-            "columns": {
-                "name": "Name",
-                "slug": "Alias",
-                "unmute": "Entsperren"
-            },
-            "empty": "Bislang hast du niemanden stummgeschaltet.",
-            "how-to": "Du kannst andere Benutzer auf deren Profilseite über das Inhaltsmenü stummschalten.",
-            "mute": "Stumm schalten",
-            "unmute": "Stummschaltung aufheben",
-            "unmuted": "{name} ist nicht mehr stummgeschaltet"
-        },
-        "privacy": {
-            "name": "Privatsphäre",
-            "make-shouts-public": "Teile von mir empfohlene Artikel öffentlich auf meinem Profil",
-            "success-update": "Privatsphäre-Einstellungen gespeichert"
+          "name": "Stummgeschaltete Benutzer",
+          "explanation": {
+            "intro": "Wenn ein anderer Benutzer von dir stummgeschaltet wurde, dann passiert folgendes:",
+            "your-perspective": "In deiner Beitragsübersicht tauchen keine Beiträge der stummgeschalteten Person mehr auf.",
+            "search": "Die Beiträge von stummgeschalteten Personen verschwinden aus deinen Suchergebnissen."
+          },
+          "columns": {
+            "name": "Name",
+            "slug": "Alias",
+            "unmute": "Entsperren"
+          },
+          "empty": "Bislang hast du niemanden stummgeschaltet.",
+          "how-to": "Du kannst andere Benutzer auf deren Profilseite über das Inhaltsmenü stummschalten.",
+          "mute": "Stumm schalten",
+          "unmute": "Stummschaltung aufheben",
+          "unmuted": "{name} ist nicht mehr stummgeschaltet"
+        },
+        "blocked-users": {
+          "name": "Blocked users",
+          "explanation": {
+            "intro": "Wenn ein anderer Benutzer von dir blockiert wurde, dann passiert folgendes:",
+            "your-perspective": "In deiner Beitragsübersicht tauchen keine Beiträge der blockierten Person mehr auf.",
+            "their-perspective": "Umgekehrt das gleiche: Die blockierte Person sieht deine Beiträge auch nicht mehr in ihrer Übersicht.",
+            "search": "Die Beiträge von blockierten Personen verschwinden aus deinen Suchergebnissen.",
+            "notifications": "Von dir blockierte Personen erhalten keine Benachrichtigungen mehr, wenn sie in deinen Beiträgen erwähnt werden.",
```

Comment:
```suggestion
```

Reply:
you don't receive notifications if there exists a `BLOCKED` relationship between two users..
this is following `mastodon`'s specifications of blocking